### ISSUE TITLE
Issue while uploading media as a comment on activity when upload terms and direct upload are enabled.

### ISSUE THREAD

**Issue by kjnanda** [rtMediaWP/rtMedia#1359]
While trying to upload the media, it takes the user where the error is thrown. Even if the user doesn't check the box it would allow uploading the media in a comment.

Please refer to this visual. https://drive.google.com/file/d/1Flyoy4Q3ePSJU_0cvwxZ5VhMM7QQjVdf/view

**Note: even if I click on upload terms checkbox I have to click on Post button to upload the media in a comment. Ideally, we should skip checking this condition while uploading media as a comment.**

-----

```
Test Environment 

WP 5.2.1
BuddyPress 4.3.0
rtMedia 4.5.6

```

**Comment by surajkrsingh** [comment]
PR:-https://github.com/rtMediaWP/rtMedia/pull/1384

**Comment by kjnanda** [comment]
This is working as expected and tested with `pre-release` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `pre-release` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `dev` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `master` branch. ✅

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V 4.5.8
